code/processes/tailer.q

```diff
@@ -1,9 +1,15 @@
+.proc.loadf [getenv[`KDBCODE],"/wdb/common.q"]                                      /-load common wdb parameters & functions
+
+upd:.wdb.replayupd;                                                                 /-start up tailer process, with appropriate upd definition
+.wdb.startup[];
+upd:.wdb.upd;
+
 \d .wdb
-.proc.loadf [getenv[`KDBCODE],"/processes/wdb.q"]
+
 hdbsettings[`taildir]:getenv`KDBTAIL
 .ds.lasttimestamp:.z.p-.ds.periodstokeep*.ds.period
 
-if[not .ds.datastripe;.lg.e[`load;"datastiping not enabled"]]                       /-errors out of tailer if datastriping is not turned on
+if[not .ds.datastripe;.lg.e[`load;"datastriping not enabled"]]                      /-errors out of tailer if datastriping is not turned on
```

Comment:
think this should be moved down a bit, past the `.tailer` namespace stuff - atm the script switches to .wdb, then to .tailer, then back to .wdb, think it'd be a bit neater if each namespace was only switched to once

Reply:
I agree that the redundant namespace switching is ugly - on reflection, I should be able to move `hdbsettings[taildir]` & `.ds.lasttimestamp` definitions down to the second instance of `\d .wdb`; and the checks on `.ds.datastripe` should be able to run in the default namespace. This would allow me to remove this first instance of `\d .wdb` without issue.

Reply:
Yes, proposed changes appear to work without issue - I've pushed a new commit for them.